domain/consensus/consensus.go

```diff
@@ -10,6 +10,17 @@ type Consensus interface {
 	BuildBlock(coinbaseData *externalapi.DomainCoinbaseData, transactions []*externalapi.DomainTransaction) (*externalapi.DomainBlock, error)
 	ValidateAndInsertBlock(block *externalapi.DomainBlock) error
 	ValidateTransactionAndPopulateWithConsensusData(transaction *externalapi.DomainTransaction) error
+
```

Comment:
What about SelectedTipBlueScore?

Reply:
The blue score is hidden in the coinbase of the selected tip block.